app/assets/stylesheets/application.scss

```diff
@@ -96,19 +91,21 @@ ul {
 }
 
 .profile-photo {
-  height: 50px;
-  margin: -5px;
   margin-right: 5px;
   width: 50px;
 }
 
+.category-header {
+  font-weight: bold;
+}
+
 div .search-bar-only {
   background-color: rgba(60, 59, 59, 0.3);
   border-color: $white;
   -moz-border-radius: 15px;
   -webkit-border-radius: 15px;
   border-radius: 15px;
-  color: $white;
+  color: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.